app/serializers/tool_group_serializer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class ToolGroupSerializer < ActiveModel::Serializer
+  attributes :id, :name
+  attribute :suggestions_weight, key: "suggestions-weight"
+
+  type "tool-group"
+
+  has_many :rule_languages, key: "rules-language"
+  has_many :rule_countries, key: "rules-country"
+  has_many :rule_praxes, key: "rules-praxis"
+  has_many :resource_tool_groups, key: "resource-tool-groups"
+  has_many :resources, through: :resource_tool_groups
```

Comment:
we don't need to expose this directly through the serializer, exposing the intermediary relationship is enough

Reply:
@cesarediaz this still needs to be adjusted, but it's a simple change :)